main.go

```diff
@@ -100,6 +101,11 @@ func templateCommand() {
 
 		for _, r := range rs.Resources {
 			fmt.Fprintf(os.Stderr, "Rendered file %s/%s:\n", rs.Name, r.Filename)
+			if *templateOutputDir != "" {
+				os.MkdirAll(*templateOutputDir, 0777)
+				f, _ := os.Create(*templateOutputDir + "/" + r.Filename)
+				fmt.Fprintf(f, r.Rendered)
+			}
```

Comment:
I don't think it should write the files *and* print them to `stdout`, so I'd prefer if this instead had an `else` clause for the "default" case of just printing.